database/types/schedule.go

```diff
@@ -99,6 +103,7 @@ func (s *Schedule) ToAPI() *api.Schedule {
 	schedule.SetScheduledAt(s.ScheduledAt.Int64)
 	schedule.SetBranch(s.Branch.String)
 	schedule.SetError(s.Error.String)
+	schedule.SetNextRun(nextTime.Unix())
```

Comment:
if `gronx.NextTickAfter` returns err does this line panic?

Reply:
ah okay cool. good question, i guess we should set it to zero in the event of an error to indicate theres no valid next_run_time?

Reply:
Yeah we can do that